exampleCourse/questions/element/richTextEditor/question.html

```diff
@@ -52,6 +52,22 @@
   
   <hr>
 
+  <pl-question-panel>
+    <p>A character/word counter can be turned on by setting <code>counter="character"</code>/
+    <code>counter="word"</code> respectively. The default setting, <code>counter="none"</code>
+    disables the counter.</p>
+
+    <pl-rich-text-editor file-name="word-count-demo" source-file-name="woodchuck-initial.html" counter="word">
+    </pl-rich-text-editor>
+
+    <hr>
+
+    <pl-rich-text-editor file-name="character-count-demo" source-file-name="woodchuck-initial.html" counter="character">
+    </pl-rich-text-editor>
+  </pl-question-panel>
+
+  <hr>
+
```

Comment:
Instead of two editors, consider adding one, then using the other counter option in combination with one of the other examples. Also, make sure the `file-name` has an extension, preferably html.